packages/itwinui-react/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 3.16.6
+
+### Patch Changes
+
+- [#2411](https://github.com/iTwin/iTwinUI/pull/2411): Fixed `Panels.Trigger` not working in older React versions where `useId` is not available.
+- [#2407](https://github.com/iTwin/iTwinUI/pull/2407): Made the light theme's text selection background opacity more consistent with that of the OS. This also affects some other opacities of other components (e.g. shadows and hover backgrounds).
```

Comment:
This is a nit that I should have mentioned in #2407, but the main reason for increasing the opacity is *not* to make it consistent with "the OS"; it's solely to increase contrast.

That it more closely matches macOS in the default accent is a good sign, but is largely irrelevant and does not need to be mentioned, especially since most end users are not going to be on macOS.